src/runtime/storage/store-interface.ts

```diff
@@ -171,13 +157,21 @@ export abstract class ActiveStore<T extends CRDTTypeRecord>
     };
   }
 }
-export abstract class ActiveMuxer<T extends CRDTTypeRecord> implements StorageCommunicationEndpointProvider<T>, AbstractActiveStore<T> {
+export abstract class ActiveMuxer<T extends CRDTTypeRecord> implements StorageCommunicationEndpointProvider<T>, ActiveStore<T> {
+  readonly storageKey: StorageKey;
+  exists: Exists;
+  readonly type: CRDTTypeRecordToType<T>;
+  readonly mode: StorageMode;
   abstract versionToken: string;
   readonly baseStore: AbstractStore;
   abstract readonly stores: Dictionary<StoreRecord<T>>;
 
   // TODO: Lots of these params can be pulled from baseStore.
   constructor(options: StoreConstructorOptions<T>) {
+    this.storageKey = options.storageKey;
+    this.exists = options.exists;
+    this.type = options.type;
+    this.mode = StorageMode.Backing;
     this.baseStore = options.baseStore;
   }
```

Comment:
you can just remove this constructor and local variables use the ActiveStore ones.
(ie remove storageKey, exists, type, mode, versionToken and baseStore). Also remove the constructor from here so the one from ActiveStore is used.

Reply:
Done.